src/cljs/ataru/hakija/application_form_components.cljs

```diff
@@ -665,6 +665,8 @@
 (defn- single-choice-option [option parent-id field-descriptor question-group-idx languages use-multi-choice-style? verifying?]
   (let [cannot-edit?   (subscribe [:application/cannot-edit? (keyword (:id field-descriptor))])
         label          (util/non-blank-val (:label option) @languages)
+        uncertain?     (and (-> field-descriptor :params :selection-group-id)
+                            @(subscribe [:state-query [:application :selection-over-network-uncertain?]]))
```

Comment:
Täs syntyy ehdollinen `subscibe` mikä on vaarallinen. Jos kääntää `and` parametrit toisin päin niin pitäs olla turvallinen.